src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -238,6 +239,14 @@ class ReimbursementAccountPage extends React.Component {
                             achData={this.props.reimbursementAccount.achData}
                         />
                     )}
+                    <ConfirmModal
+                        title={this.props.translate('companyStep.confirmModalTitle')} // TODO: Update translation key
+                        onConfirm={() => hideErrorModal()}
+                        prompt={this.props.reimbursementAccount.errorModalMessage || this.props.translate('companyStep.confirmModalPrompt')}
+                        isVisible={Boolean(this.props.reimbursementAccount.isErrorModalVisible)}
```

Comment:
Any reason to do a `Boolean()` cast here?

Reply:
The property `this.props.reimbursementAccount.isErrorModalVisible` is undefined at the start, so you get some errors in the console if you pass undefined to `isVisible'

![Screen Shot 2021-08-30 at 4 07 43 PM](https://user-images.githubusercontent.com/87341702/131416966-4f7763c7-3492-41a3-a334-94e96c7048fe.png)

Reply:
Ah maybe give it a default value? If it's `undefined` then pass `false` ?

Reply:
you mean like `isVisible={this.props.reimbursementAccount.isErrorModalVisible || false}`, or setting the default as default prop in ConfirmModal.js? If it is the second, the linter complains because it doesn't like default props for 'required props'. If it is the first... I guess it is almost the same.

Linter: `propType "isVisible" is required and should not have a defaultProps declaration`

Reply:
>If it is the first... I guess it is almost the same.

Fair enough, I've seen it both ways, but mostly have seen use of `lodashGet()` when needing a default value.

Reply:
> > If it is the first... I guess it is almost the same.
> 
> Fair enough, I've seen it both ways, but mostly have seen use of `lodashGet()` when needing a default value.

Hmm you are right, lodashGet seems to be used a lot for this in the project... I guess I'll be consistent

Reply:
Ok I could call this a NAB (not a blocker) too - seems minor looking again today ;)